src/lib/main.js

```diff
@@ -175,4 +178,23 @@ module.exports = async (argv, env) => {
   }
 };
 
-const executeAction = action => require(`../fn/${action}`)();
+
+const requiresInstance = actions => actions.find(actionName => {
+  const actionLib = require(`../fn/${actionName}`);
+  if (typeof actionLib === 'function') {
+    return true;
+  } else if (typeof actionLib.requiresInstance === 'boolean') {
```

Comment:
the `else` is not required after `if() { return }` blocks

Reply:
Yeah I know, but IMO when all branches are "logically equal" (like one isn't an edge case or whatever, they're all valid) it looks clearer to have the return statements lined up vertically, as it puts those forks mentally on an equal footing. I realise this is A E S T H E T I C S territory, let me know if you're strongly against the redundancy.

Reply:
This is actually one of the things we call out explicitly in our style guide. https://github.com/medic/medic-docs/blob/master/development/style-guide.md#return-early-from-functions

Reply:
:+1: